ext/sqlite3/php_sqlite3_structs.h

```diff
@@ -62,7 +62,7 @@ typedef struct _php_sqlite3_collation {
 
 /* Structure for SQLite Database object. */
 typedef struct _php_sqlite3_db_object  {
-	int initialised;
+	bool initialised;
 	sqlite3 *db;
 	php_sqlite3_func *funcs;
 	php_sqlite3_collation *collations;
```

Comment:
Shouldn't you move this next to the `exception` field to back it? As there are going to be 7 bytes of padding, and then again a bunch of padding after the `exception` field.